runtime/doc/editing.txt

```diff
@@ -1236,29 +1244,50 @@ present in 'cpoptions' and "!" is not used in the command.
 							*:chd* *:chdir*
 :chd[ir][!] [path]	Same as |:cd|.
 
+							*:tc* *:tcd*
+:tc[d][!] {path}        Like |:cd|, but set the current directory for the
+                        current tab.  The current directory for other tabs
+                        or the local directory of windows in this tab is not
+                        changed, except for the current window.
+
+							*:tcd-*
+:tcd[!] -		Change to the previous current directory (before the
+			previous ":tcd {path}" command).
+
+							*:tch* *:tchdir*
+:tch[dir][!]		Same as |:tcd|.
+
 							*:lc* *:lcd*
 :lc[d][!] {path}	Like |:cd|, but only set the current directory for the
 			current window.  The current directory for other
-			windows is not changed.
+			windows or any tabs is not changed.
```

Comment:
I don't think using numbers makes sense. A letter prefix is more intuitive and that's exactly what I did: `t` for tab, `l` for window and nothing for global. I think `wcd` would be a better choice than `lcd`, but that's the way it is (we could make an alias, but I don't think we need yet another command).

Or did you mean like a bitmask? 0 for global, 1 for window, 2 for tab and 3 for window and tab? That would be superfluous, a `:tcd` already overwrites the window-local working directory and a `:cd` overwrites both.